app/moves/middleware/set-results.moves.test.js

```diff
@@ -174,6 +173,49 @@ describe('Moves middleware', function () {
       })
     })
 
+    context('with `personEscortRecordFeature` disabled', function () {
```

Comment:
I didn't see a test for this feature being disabled, so it seemed like a good idea to add one now.